src/main/java/org/folio/circulation/domain/RequestQueue.java

```diff
@@ -18,7 +19,10 @@ public RequestQueue(Collection<Request> requests) {
 
     // Ordering requests by position, so we can add and remove them
     // without sorting and just re-sequence from 1 to n
-    this.requests.sort(Comparator.comparingInt(Request::getPosition));
+    this.requests.sort(Comparator
+      .comparingInt(request -> Optional.ofNullable(request.getPosition())
+        .orElse(0)
+      ));
```

Comment:
How was this change identified? Can the circumstance be demonstrated with a test?

Reply:
this is an emergency check :) Cate mentioned this on https://issues.folio.org/browse/UIREQ-386.

Not sure how we can test it. Maybe just unit test.

Reply:
Can we create a request without a position in storage, to mimic the description you gave for the pull request?